hermes/src/lib.rs

```diff
@@ -183,7 +188,7 @@ pub trait TtsBackendFacade: ComponentBackendFacade {
 pub trait NluFacade: ComponentFacade {
     fn publish_query(&self, query: NluQueryMessage) -> Fallible<()>;
     fn publish_partial_query(&self, query: NluSlotQueryMessage) -> Fallible<()>;
-    fn publish_reload(&self) -> Fallible<()>;
+    fn publish_reload(&self, reload: LoadMessage) -> Fallible<()>;
```

Comment:
use a separate `ReloadRequestMessage`, with a single field, `id` the `component` in `LoadMessage` is useless and missleading here